src/AdminConsole/Components/Routes.razor

```diff
@@ -1,13 +1,18 @@
 @using Microsoft.AspNetCore.Components.Authorization
 @using Passwordless.AdminConsole.Components.Shared
+
+@inject IHttpContextAccessor HttpContextAccessor
+
 <Router AppAssembly="@typeof(Program).Assembly">
     <Found Context="routeData">
-        <AuthorizeRouteView RouteData="@routeData" DefaultLayout="@typeof(Layouts.MainLayout)">
+        <!-- The 'Resource' property will be used in 'HasAppHandler' to obtain the 'app' route parameter -->
+        <!-- If we implement new policies to protect resources, we should protect them with resource based uri and compare them with the user's claims. -->
+        <AuthorizeRouteView RouteData="@routeData" Resource="@HttpContextAccessor.HttpContext" DefaultLayout="@typeof(Layouts.MainLayout)">
```

Comment:
`HasAppHandler` reads the `app` route parameter from the HttpContext. This strategy will leave us with maximum flexibility for new policies.